vspace/src/main/java/edu/asu/diging/vspace/core/data/SlideExternalLinkRepository.java

```diff
@@ -0,0 +1,19 @@
+package edu.asu.diging.vspace.core.data;
+
+import java.util.List;
+
+import org.javers.spring.annotation.JaversSpringDataAuditable;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+
+import edu.asu.diging.vspace.core.model.IExternalLinkSlide;
+import edu.asu.diging.vspace.core.model.impl.ExternalLinkSlide;
+
+@Repository
+@JaversSpringDataAuditable
+public interface SlideExternalLinkRepository extends PagingAndSortingRepository<ExternalLinkSlide, String> {
+
+    public List<IExternalLinkSlide> findBySlide_Id(String slideId);
+
+    public void deleteByExternalLink(IExternalLinkSlide link);
```

Comment:
is this used anywhere?

Reply:
currently, I do not use `deleteByExternalLink` anywhere. It was added by Swetalina and included in her tests. I use `findBySlide_Id` to get the list of External Links in the Slide controllers.

Reply:
I can hardly see a use case, where this would be useful.

Reply:
are you talking about the repo as a whole or just `deleteByExternalLink`?

Reply:
just `deleteByExternalLink`. I don't think we need this method.